mw-webapp/src/logic/userPage/UserPage.tsx

```diff
@@ -1,22 +1,38 @@
-
-import {Navigate, useParams} from "react-router-dom";
+import {Navigate, Params, useParams} from "react-router-dom";
+import {Button} from "src/component/button/Button";
 import {HeadingLevel, Title} from "src/component/title/Title";
+import {WayPreviewDAL} from "src/dataAccessLogic/WayPreviewDAL";
 import {OwnWaysTable} from "src/logic/waysTable/OwnWaysTable";
 import {pages} from "src/router/pages";
 import styles from "src/logic/userPage/UserPage.module.scss";
 
+/**
+ * Query param types
+ */
+interface QueryParamTypes extends Params {
```

Comment:
It's better to keep url lprams for pages near pages

Reply:
Moved to separate file